splash-screen/src/web.ts

```diff
@@ -0,0 +1,17 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type {
+  SplashScreenPlugin,
+  SplashScreenShowOptions,
+  SplashScreenHideOptions,
+} from './definitions';
+
+export class SplashScreenWeb extends WebPlugin implements SplashScreenPlugin {
+  async show(_options?: SplashScreenShowOptions): Promise<void> {
+    return undefined;
```

Comment:
```suggestion
    throw this.unimplemented('Not implemented for web.');
```

Reply:
it was like that, so I didn't want to change the behavior